cli/dcoscli/package/main.py

```diff
@@ -239,9 +228,9 @@ def _describe(package_name,
     :rtype: int
     """
 
-    if package_versions and \
-            (app or cli or options_path
-             or render or package_version or config):
+    if (package_versions and
+        (app or cli or options_path or
+         render or package_version or config)):
```

Comment:
My linter complains with the previous formatting. Multi-line expressions need to end with a binary operator.